test/unit/pipeline/test_train_model.py

```diff
@@ -140,6 +142,32 @@ def test_train_model_pipeline_young_model(self, serializer_mock, pipeline_mock):
         )
         self.assertFalse(pipeline_mock.called)
 
+    def test_feature_importance(self):
```

Comment:
Good idea to add a unit test for the feature importance functionality!
I think currently this is only tested for a single machine learning model type, correct?
In fact, looking at this test, I think the training pipeline altogether is only tested for a single model type. 

Perhaps you could change the test so that the `test_train_model_pipeline_core_happy_flow` iterates over all model types specified in `openstf.enums.MLModelType`? You could than add the test of `model.get_feature_importance` there as well, instead of training a model specifically for that.

Reply:
test_train_model_pipeline_core_happy_flow gets the pj from prediction_jobs.json. In this file we only have a xgb model. And since it gets the model type from the pj I can't loop it.

Reply:
Let's have a look at it together, I have good hopes that we can make it work :)